### DISCUSSION TITLE
compose手动增加inception容器后，只有http登陆页面，并且密码方式无法进入

### DISCUSSION THREAD

**Discussion by hydgladiator** [hhyo/Archery#2098] [Q&A]
![图片](https://user-images.githubusercontent.com/13326206/228398539-e2e5b5e9-72a3-423e-bf17-c633c21355c7.png)
1.9.1版本，原来的https页面可以登陆，并且能够正常使用，发现提交SQL缺少检查机制的inception，发现配置项无inception，手动增加inception后，compose重启后，出现如标题现象。
去掉inception，compose重启，现象依旧，无法回到原有正常登陆方式。

**Comment by hhyo** [comment]
可以尝试销毁重新部署...

**Comment by hydgladiator** [comment]
docker stop archery 
docker rm archery 
docker-compose down
docker-compose -f docker-compose.yml up -d
docker-compose ps 
![图片](https://user-images.githubusercontent.com/13326206/233319657-98826011-166f-43a3-b2e7-1e9674380485.png)

以上步骤手动销毁archery容器后，并重新构建了一次，日志都正常，但是还是一样的登陆状态

**Reply by hhyo** [reply]
#2121
